lib/instrumentation/utils.ts

```diff
@@ -0,0 +1,11 @@
+export function isTracingEnabled(): boolean {
+  return isTraceDebuggingEnabled() || isTraceSendingEnabled();
+}
+
+export function isTraceDebuggingEnabled(): boolean {
+  return !!process.env.RENOVATE_DEBUG_TRACING;
```

Comment:
This name may not be descriptive enough if its only use is for OTEL/instrumentation. e.g. people mix it up with LOG_LEVEL=debug

Reply:
I have changed it to `RENOVATE_TRACING_CONSOLE_EXPORTER`, this should clear it up a bit.